pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -121,7 +121,11 @@ func (lb *ListenerBuilder) buildVirtualOutboundListener() *ListenerBuilder {
 
 	filterChains := buildOutboundCatchAllNetworkFilterChains(lb.node, lb.push)
 
-	actualWildcard, _ := getActualWildcardAndLocalHost(lb.node)
+	var actualWildcard, extrActualWildcard string
```

Comment:
this seems to complex and error prone and duplicated. Can we refactor this a bit to make it easier to use?

Reply:
Sure, will refactor them

Reply:
done, mark them with sufix `IPv4` and `IPv6`